src/CMakeLists.txt

```diff
@@ -871,16 +871,14 @@ add_subdirectory(compressor)
 
 add_subdirectory(tools)
 
-# dmClock
-
-add_subdirectory(dmclock) # after gmock
-add_dependencies(tests dmclock-tests dmclock-data-struct-tests)
+# dmClock (after gmock)
 
+add_subdirectory(dmclock/src)
 if(WITH_TESTS)
-  install(PROGRAMS
-    ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/dmclock-tests
-    ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/dmclock-data-struct-tests
-    DESTINATION bin)
+  # note: add_test is not being called, so dmclock tests aren't part
+  # of ceph tests
+  add_subdirectory(dmclock/test)
```

Comment:
ah, I see that "add_subdirectory(dmclock)" has been dropped, but we still have "add_subdirectory(dmclock/test)" which causes cmake to run https://github.com/ivancich/ceph-fork/blob/wip-bring-in-dmclock-p2-gtest/src/dmclock/test/CMakeLists.txt - or am I missing something?